resources/lang/en/profile.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    "profile_updated" => "Successfully updated profile data!",
+    "password_updated" => "Successfully updated password!",
+    "notification_new_email" => "Notification successfully sended for your new email! Please confirm on new email your email address.",
```

Comment:
Isn't `sent` more accurate past tense of `send` than `sended`? Also check other files, please
```suggestion
    "notification_new_email" => "Notification about your new e-mail has been sent! Please confirm your new e-mail address.",
```